runtime/doc/usr_05.txt

```diff
@@ -378,17 +373,102 @@ The generic names for the filetype plugins are: >
 Here "<name>" can be any name that you prefer.
 Examples for the "stuff" filetype on Unix: >
 
-	~/.vim/ftplugin/stuff.vim
-	~/.vim/ftplugin/stuff_def.vim
-	~/.vim/ftplugin/stuff/header.vim
+	~/.nvim/ftplugin/stuff.vim
+	~/.nvim/ftplugin/stuff_def.vim
+	~/.nvim/ftplugin/stuff/header.vim
 
 The <filetype> part is the name of the filetype the plugin is to be used for.
 Only files of this filetype will use the settings from the plugin.  The <name>
 part of the plugin file doesn't matter, you can use it to have several plugins
 for the same filetype.  Note that it must end in ".vim".
 
+USING PLUGIN BUNDLES                                      *plugin-bundles*
+
+The manual method described above has the problem that things can get messy
+once you install a few plugins, specially if they include more than one file
+(which is common).  A way to solve this is to install plugins in their own
+directories and then add the path of those to the 'runtimepath' setting (so
+Vim will use the files from those directories when looking for plugins).  In
+practice, this means that instead of having a |vimfiles| tree like >
+
+	~.nvim/
+	    plugin/
+		a.vim
+	    ftplugin/
+		b.vim
+		e.vim
+	    syntax/
+		b.vim
+
+you can have something like this >
+
+	~.nvim/
+	    bundles/
+		a/
+		    plugin/
+			a.vim
+		b/
+		    ftplugin/
+			b.vim
+		    syntax/
+			b.vim
+		e/
+		    ftplugin/
+			e.vim
+
+This method allows the plugins to be distributed as their own trees, so they
+can be installed without affecting other plugins.  This is very helpful when
+synchronizing with a version control repository.
+
+So supposing the user wants to install a plugin that's distributed as this tree: >
+
+	autoload/
+	    awesome.vim
+	plugin/
+	    awesome.vim
+	doc/
+	    awesome.txt
+
+from http://github.com/developer/awesome-vim, they can do so by (assuming there
+is no "bundles" directory)
+
+    1. creating the plugin directory >
+
+	$ mkdir -p ~/.nvim/bundles/awesome-vim
+
+<    2. copying the files into the "bundles/awesome-vim/" folder >
+
+	$ cp awesome-vim/* ~/.nvim/bundles/awesome-vim/
+
+<    3. adding >
+
+	set runtimepath+=~/.nvim/bundles/awesome-vim
+
+<       to their |vimrc|.
+
+Most commonly, instead of steps 1-2, this should work: >
```

Comment:
Adverb which modifies what? In this position with these commas this looks like an introductory phrase (where adverb has nothing to modify because this is rather independent) and not something that modifies verb “work”. I would rephrase this as “Most likely steps 1 and 2 can be replaced with” because I am pretty sure “instead of steps 1-2” could not be placed in that position.